draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -201,21 +199,16 @@ multiple paths. In addition, the multipath extensions enable to tradeoff timelin
 which is important for low-latency applications that do not require
 guaranteed delivery services, such as Audio/Video streaming.
 
-MP-DCCP was first suggested
-in the context of the 3GPP work on 5G multi-access solutions
-{{I-D.amend-tsvwg-multipath-framework-mpdccp}} and for hybrid access
-networks {{I-D.lhwxz-hybrid-access-network-architecture}}{{I-D.muley-network-based-bonding-hybrid-access}}, where
-MP-DCCP can be applied for load-balancing, seamless session handover, and bandwidth
-aggregation (referred to as Access Traffic Steering, Switching, and Splitting (ATSSS)
-in the 3GPP terminology {{TS23.501}}).
+In addition to the integration into DCCP services, other use cases of MP-DCCP are the support of
+3GPP 5G multi-access solutions {{I-D.amend-tsvwg-multipath-framework-mpdccp}} or hybrid access networks
+{{I-D.lhwxz-hybrid-access-network-architecture}}{{I-D.muley-network-based-bonding-hybrid-access}}, in
+which MP-DCCP can be used for load balancing, seamless session handover and bandwidth aggregation
+(referred to as Access Traffic Steering, Switching, and Splitting (ATSSS) in 3GPP terminology {{TS23.501}}).
 More details on potential use cases for MP-DCCP are provided in {{multipath-dccp.org}}, {{IETF115.Slides}}, and {{MP-DCCP.Paper}}.
 All these use cases profit from an Open Source Linux reference implementation provided under {{multipath-dccp.org}}.
 
-Encapsulation for DCCP in UDP is defined in {{RFC6773}}.
-{{I-D.amend-tsvwg-multipath-framework-mpdccp}} proposes a 
-lightweight encapsulation for DCCP flow headers appropriate for unreliable 
-IP traffic in terms of UDP and other non-TCP packets in comparison to MPTCP.
-This is not considered in the present specification. 
+The encapsulation of non-DCCP traffic (e.g. UDP or IP) in MP-DCCP to enable the above-mentioned use cases is not considered in this specification and is the subject of {{I-D.amend-tsvwg-multipath-framework-mpdccp}}.
+Similarly, the encapsulation of DCCP traffic in UDP is not discussed in detail in this document. This method can be used to pass middleboxes that do not support DCCP and is defined in {{RFC6773}} or is considered in {{I-D.amend-tsvwg-dccp-udp-header-conversion}} to achieve the same with less overhead.
```

Comment:
I don't parse the last sentence.

Reply:
@boucadair , can you check if the changes in https://github.com/tsvwg/ietf-multipath-dccp/pull/295/commits/6cf1107b7605e38fb221d4c45ca8e43dd9617940 provide clarification?